mgmtsystem_nonconformity_quality_control_oca/tests/test_inspection.py

```diff
@@ -0,0 +1,61 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo.tests import common
+
+logger = logging.getLogger(__name__)
+
+
+class TestQualityControl(common.TransactionCase):
+    def setUp(self):
```

Comment:
```suggestion
    def setUpClass(cls):
```
(and replace all occurrences of `self` with `cls` within this method)



Keep in mind `setUp` runs before every test.
whilst `setUpClass` runs only once for all tests, so it's more performant

Reply:
I tried but pre-commit dislike it. So I changed setUp but not cls.
Do you have some other suggestion?